lib/src/over_react_test/react_util.dart

```diff
@@ -106,7 +105,7 @@ ReactElement renderShallow(ReactElement instance, {bool autoTearDown = true, Cal
 void unmount(dynamic instanceOrContainerNode) {
   if (instanceOrContainerNode == null) return;
 
-  Element containerNode;
+  late Element? containerNode;
```

Comment:
In this case, if you use `final` instead, it'll verify statically that containerNode is assigned before it's read (just like an unassigned non-nullable variable, like if it were `Element containerNode;`)
```suggestion
  final Element? containerNode;
```

Reply:
Oh nice! Thanks!